app/model/Model.scala

```diff
@@ -295,6 +300,10 @@ object JourneyData {
   implicit val confirmableAddressDetailsFormat = Json.format[ConfirmableAddressDetails]
   implicit val confirmableAddressFormat = Json.format[ConfirmableAddress]
   implicit val proposedAddressFormat = Json.format[ProposedAddress]
+  implicit val timeoutFormat: Format[Timeout] = (
+    (JsPath \ "timeoutAmount").format[Int](min(120)) and
```

Comment:
Is it really necessary for us to be this strict about the configured value? Why can't it just be a number? I would prefer it if ALF could remain as agnostic as possible on what are effectively user requirements of calling service applications.

Reply:
This is a safeguard to prevent users supplying a value that is too low some of the following could happen:
 - The JavaScript calculation relies on a positive number which is not unreasonably low
 - End users might constantly seeing the popup if the service opts to use the timeout feature

Reply:
This timeout feature is *only* for SCRC. Other client apps do not need it and will not be using it. You need to give more consideration to this fact. Your changes *must* be backward compatible. They must not require any other ALF client to make changes. The fact that you need 120 or greater is of no concern to them.